keps/0010-package-manager.md

```diff
@@ -166,12 +167,82 @@ Our official repository is hosted on Google Cloud Storage and following a flat s
 
 We rely on just an HTTP Server, e.g. the out-of-the-box solution that Google Cloud Storage provides, that serves operator `tgz` files and makes them available to users.
 
-The logic for keeping the operators in sync should live in the CLI and is not defined on this KEP . That way the HTTP server only has to serve the index and the Package `tgz` files, without having to implement any business logic. 
+The logic for keeping the operators in sync should live in the CLI and is not defined on this KEP . That way the HTTP server only has to serve the index and the Package `tgz` files, without having to implement any business logic.
 
 The proposed structure is fairly easy to replicate and highly customizable.
 
 Safety when distributing our Packages is another concern. As we continue working on this KEP we will add more details on how we can verify and prevent `Arbitrary software installation`, `Vulnerability to key compromises`, etc. For now using HTTPS to fetch the index and Packages is considered sufficient.
 
+### Index File Specification
+
+`index.yaml` is the base definition of a repository.  It follows the following reference format.
+
+```yaml
+apiVersion: str(desc='version of repository')
+entries: map[string]PackageVersions
+  str(desc='name of operator'):
+  - name:  str(max=253)
+    version: semver()
+    appVersion: str(desc='component controlled version', required=False)
+    home: url(desc='home page, git repo', required=False)
+    sources: array(str, required=False)
```

Comment:
wow, why is sources a list? Do we expect operator source to live in multiple repos? I also don't feel like this should be in index right now though

Reply:
this is already part of our code...  part of the intent of this PR was to surface all of what is current.   Is your proposal to remove from code? or to reduce to 1?  or to not expose?